src/firefly/java/edu/caltech/ipac/util/StringUtils.java

```diff
@@ -750,6 +750,21 @@ public static <R> R getSafe(FuncWithEx<R> func, R defaultVal) {
         }
     }
 
+    /**
+     * A supplier that throws exception
+     */
+    public interface TryWithEx {
+        void run() throws Exception;
+    }
+
+    public static void tryIt(TryWithEx func) {
```

Comment:
This function name is confusing. I studied it and understand now but `tryIt()` seems to vague for a general string function.

Reply:
I will remove.  I plan to introduce more functional helpers gradually in a separate util class.